polus-feature-extraction-plugin/plugin.json

```diff
@@ -96,7 +96,7 @@
       {
```

Comment:
Embedded pixel size description should read:

"Use embedded pixel size"

Then the README should explain that it will use it only if present.